Grammar (#76)
fhdsl/NIH_Data_Sharing: index.Rmd

</div>

<div class = question_box>
**Does this apply to me?** This will apply to most NIH grants that create data (regardless of funding level), although some grant mechanisms and some data are exempt. **It is not required to share all data**, however a justification is required. Reasons not to share that are acceptable may be ethical, technical, or legal.
</div>

<div class = data>
**Where should I share my data?** The NIH has lists of suggested repositories, to help you find appropriate repositories for your data. If there is not an appropriate repository, you can share your data at an institutional (if available) or generalist repository.
</div>

<div class = money>

> **How do I budget this?** You may request funds for data management and sharing as direct costs (including personnel costs). You may not infrastructure costs as direct costs, as these should be included as indirect costs.

Reviewer: ```suggestion
**How do I budget this?** You may request funds for data management and sharing as direct costs (including personnel costs). Infrastructure costs should only be included as indirect costs.
```

Author: much better - thanks!
